habitat-lab/habitat/sims/habitat_simulator/debug_visualizer.py

```diff
@@ -539,15 +593,15 @@ def _peek_bb(
             world_transform = mn.Matrix4.identity_init()
         look_at = world_transform.transform_point(bb.center())
         bb_size = bb.size()
-        fov = self.sensor._spec.hfov
+        fov = 90 if self._equirect else self.sensor._spec.hfov
```

Comment:
In the 'normal' mode, `hfov` would be the default sensor field of view?

Reply:
The DebugVisualizer adds its own sensor, so whatever that sensor's fov is will be used here.